vault/changelog.md

```diff
@@ -2,11 +2,48 @@
 id: 9bc92432-a24c-492b-b831-4d5378c1692b
 title: Changelog
 desc: ''
-updated: 1644943670603
+updated: 1645544640720
 created: 1601508213606
 nav_order: 2
 ---
 
+## 0.83.0
+
+### Breaking changes
+
+The latest release of Dendron will prompt users to automatically migrate their `dendron.yml` configurations to use the latest publishing configuration. **This update is not backwards compatible**, meaning that updated workspace configurations (`version: 5`) won't work with older versions of `dendron-cli` used for publishing.
```

Comment:
change to "won't work with older versions of `dendron-cli`". even though its for publishing, older dendron-cli will not work if you are using v5 config